samples: matter: Add ICD LIT functionality (#16392)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Matter Bridge
-------------

* Added:

  * The :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILES_PATH` Kconfig option, which specifies ZAP files location for the application.

>     By default the option points to :file:`src/default_zap` directory and can be changed to any path relative to application's location, under which :file:`.zap` file and :file:`zap-generated` directory are located.

Reviewer: ```suggestion
    By default, the option points to the :file:`src/default_zap` directory and can be changed to any path relative to application's location that contains the ZAP file and :file:`zap-generated` directory.
```

Thingy:53: Matter weather station
---------------------------------

* Added:

  * The :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILES_PATH` Kconfig option, which specifies ZAP files location for the application.

>     By default the option points to :file:`src/default_zap` directory and can be changed to any path relative to application's location, under which :file:`.zap` file and :file:`zap-generated` directory are located.

Reviewer: ```suggestion
    By default, the option points to the :file:`src/default_zap` directory and can be changed to any path relative to application's location that contains the ZAP file and :file:`zap-generated` directory.
```

Matter samples
--------------

* Added:

  * The :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILES_PATH` Kconfig option, which specifies ZAP files location for the sample.

>     By default the option points to :file:`src/default_zap` directory and can be changed to any path relative to sample's location, under which :file:`.zap` file and :file:`zap-generated` directory are located.

Reviewer: ```suggestion
    By default, the option points to the :file:`src/default_zap` directory and can be changed to any path relative to sample's location that contains the ZAP file and :file:`zap-generated` directory.
```
